include/boost/geometry/algorithms/detail/disjoint/multipoint_geometry.hpp

```diff
@@ -63,12 +63,13 @@ namespace detail { namespace disjoint
 class multipoint_multipoint
 {
 private:
-    template <typename Iterator, typename CSTag>
+    template <typename Iterator, typename Strategy>
     class unary_not_disjoint_predicate
-        : geometry::less<void, -1, CSTag>
+        : geometry::less<void, -1, Strategy>
```

Comment:
Can we call it maybe: `CompareStrategy` ?

Reply:
I do not know, I kept it that way since this is the name used by apply function (below) that actually pass that type.

Reply:
ok